lib/geocoder/lookups/db_ip_com.rb

```diff
@@ -0,0 +1,49 @@
+require 'geocoder/lookups/base'
+require 'geocoder/results/db_ip_com'
+
+module Geocoder::Lookup
+  class DbIpCom < Base
+
+    def name
+      'DB-IP.com'
+    end
+
+    def supported_protocols
+      if configuration.api_key[1]
+        [:https]
+      else
+        [:http]
+      end
+    end
+
+    def required_api_key_parts
+      ['api_key', 'is_paid']
```

Comment:
Geocoder uses use_https to set https e.g. 
```ruby
    Geocoder.configure(:ip_lookup => :ipinfo_io, :use_https => true)
```

Could you update required_api_key_parts, supported_protocols and your https test?
Thank you so much!